modules/placement-driver/src/test/java/org/apache/ignite/internal/placementdriver/leases/LeaseSerializationTest.java

```diff
@@ -17,61 +17,82 @@
 
 package org.apache.ignite.internal.placementdriver.leases;
 
-import static org.apache.ignite.internal.placementdriver.leases.Lease.fromBytes;
+import static java.nio.ByteOrder.LITTLE_ENDIAN;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
 import java.util.ArrayList;
-import java.util.List;
 import org.apache.ignite.internal.hlc.HybridTimestamp;
 import org.apache.ignite.internal.replicator.ReplicationGroupId;
 import org.apache.ignite.internal.replicator.TablePartitionId;
+import org.jetbrains.annotations.Nullable;
 import org.junit.jupiter.api.Test;
 
-/**
- * Tests for lease encoding and decoding from byte arrays.
- */
+/** Tests for lease encoding and decoding from byte arrays. */
 public class LeaseSerializationTest {
     @Test
-    public void test() {
-        Lease lease;
-
+    public void testLeaseSerialization() {
         long now = System.currentTimeMillis();
         ReplicationGroupId groupId = new TablePartitionId(1, 1);
 
-        lease = Lease.EMPTY_LEASE;
-        assertEquals(lease, fromBytes(ByteBuffer.wrap(lease.bytes()).order(ByteOrder.LITTLE_ENDIAN)));
+        checksSerialization(Lease.EMPTY_LEASE);
```

Comment:
To be honest I don't like such small utility helpers. Before it was absolutely clear that we check that lease is equals to fromBytes() and after the refactoring it's not.

Reply:
I don’t agree, it’s a matter of taste, there’s much less repetitive code.

Reply:
It's a matter of taste, right. And because of that - it's up to you here.